package.json

```diff
@@ -1,8 +1,8 @@
 {
   "name": "cactbot",
-  "version": "0.34.1",
-  "releaseSummary": "Zelenia EX, Suzaku Unreal, i18n",
-  "releaseInDraft": "true",
+  "version": "0.34.2",
+  "releaseSummary": "initial M5N, M6N, M5S, 7.2 jobs module changes, i18n",
```

Comment:
This should perhaps be "5S triggers" only, since the timeline isn't yet in place?

Reply:
"Initial" doesn't imply a timeline though, it doesn't imply anything other than first-pass effort really.